lib/src/value_layout_builder.dart

```diff
@@ -103,22 +101,18 @@ class _RenderValueLayoutBuilder<T> extends RenderBox
   void performLayout() {
     final BoxConstraints constraints = this.constraints;
     rebuildIfNecessary();
-    if (child != null) {
-      child.layout(constraints, parentUsesSize: true);
-      size = constraints.constrain(child.size);
-    } else {
-      size = constraints.biggest;
-    }
+    child?.layout(constraints, parentUsesSize: true);
+    size = constraints.constrain(child?.size ?? constraints.biggest);
```

Comment:
I prefer that we keep the previous if/else branches. In the case of the null child, we would constrain a size already constrained by doing this, and it would be less performant.